app/celery/provider_tasks.py

```diff
@@ -145,6 +145,7 @@ def deliver_letter(self, notification_id):
             ) from e
     except Exception as e:
         if isinstance(e, DvlaDuplicatePrintRequestException):
```

Comment:
can make this even tidier by moving it to a `except DvlaDuplicatePrintRequestException` handler